larky/src/main/java/com/verygood/security/larky/ModuleSupplier.java

```diff
@@ -20,6 +20,7 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 
+import com.verygood.security.larky.modules.AccountUpdaterModule;
```

Comment:
This does not belong in the general modules package. This belongs in the `vgs` package.